### PR TITLE
Keycard landing page and FAQ page.

### PR DESCRIPTION
**Pull request by NistorCristian**
I've created a new landing page for Keycard and a general FAQ page. The Keycard page requested some nice animations and we would like, if everything is ok in this PR, to push into develop so that @j-zerah can give his feedback on these 2 pages.

Thanks

Signed-off-by: Nistor Cristian <contact@nistorcristian.com>

### CODE REVIEW

**Comment by jakubgs** [comment]
You know, you can use something like https://ngrok.com/ to do demos of pages.
Here's a demo of this branch: https://206098d5b59a.eu.ngrok.io/
If this looks good to you @j-zerah please say so.

**Comment by hesterbruikman** [comment]
@NistorCristian could you please point me to the content file for the new FAQ? Can't find it 😬. Would like to make some minor copy edits

**Comment by jakubgs** [comment]
@hesterbruikman it looks like it's all held within the english language translation file:
https://github.com/status-im/status.im/blob/73c59fc9233e0da7f8e0e2e73a28452c9168baef/themes/navy/languages/en.yml#L597-L607
Not a fan of that solution, but I guess if we intend to translate those then there's no other way.

**Comment by jakubgs** [comment]
@NistorCristian can you rebase on `develop`?

**Comment by StatusSceptre** [comment]
@jakubgs - can you deploy these to develop. I cant seem to review the pages with ngrok anymore

**Comment by jakubgs** [comment]
@NistorCristian you did not `rebase` the branch, you `merge`d it into `develop`, that's wrong:
```
This branch cannot be rebased due to conflicts
```
I try to rebase myself but I see conflicts in `themes/navy/source/scss/main.scss` that I don't know how to resolve.

File: themes/navy/source/scss/main.scss [hunk 1]
```diff
@@ -390,36 +390,30 @@ h4{
     justify-content: center;
 }
 
-.p-t-160{
-    padding-top: 160px;
-}
-.p-b-160{
-    padding-bottom: 160px;
+@for $i from 0 through 20 {
```
Comment (jakubgs) [inline themes/navy/source/scss/main.scss:393]: This damned file is getting out of control. 6228 lines, absolutely insane. We can't just keep adding stuff to it.
There's 6228 lines and we use `@extend` only __once__, here:
https://github.com/status-im/status.im/blob/5620c93d95d8707d46f171135599187cf6af2912/themes/navy/source/scss/main.scss#L2756
This file will be serious trouble in the future if we let it grow like that.

**Comment by jakubgs** [comment]
I've rebased the `themes/navy/source/scss/main.scss` file myself. Was a bit tricky because of changes by Jinho in dd27c687, but I __think__ it's correct.

**Comment by jzerah** [comment]
It's hard for me to determine what exactly is needed. Perhaps @jinhojang6 can look into this and refactor the CSS.

If not, we can pause all web updates after the next big push (July) and refactor.

**Comment by jinhojang6** [comment]
Let me look into it after the major updatea in July @jakubgs

**Pull request merged**